### PR TITLE
Package names must be in nevra format

### PR DESCRIPTION
**Pull request by daviddavis**

### CODE REVIEW

File: rpm/assets/modules.yaml [hunk 1]
```diff
@@ -16,7 +16,7 @@ data:
         - walrus
   artifacts:
     rpms:
-      - walrus-5.21-1.noarch
+      - walrus-0:5.21-1.noarch
```
Comment (Ichimonji10) [inline rpm/assets/modules.yaml]: Isn't an exclamation point supposed to separate the epoch from the remainder of the version string?

Comment (daviddavis) [reply rpm/assets/modules.yaml]: From googling, it looks like it's suppose to be `name-epoch:version-release.architecture` which repoquery also seems to confirm:

```
$ repoquery --nevra '*vim*'
vim-enhanced-2:7.4.160-4.el7.x86_64
```

Comment (Ichimonji10) [reply rpm/assets/modules.yaml]: TIL that there are multiple ways to separate an epoch from the rest of the version. See [PEP 440 - Version epochs](https://www.python.org/dev/peps/pep-0440/#version-epochs).

Comment (daviddavis) [reply rpm/assets/modules.yaml]: Ah interesting.

**Pull request merged**